volcano.tf

```diff
@@ -0,0 +1,66 @@
+################################################################################
+# Volcano Scheduler for TorchX
+################################################################################
+
+resource "helm_release" "volcano" {
+  count = var.enable_volcano ? 1 : 0
+
+  name                       = try(var.volcano_helm_config["name"], "volcano")
+  repository                 = try(var.volcano_helm_config["repository"], "https://github.com/volcano-sh/volcano.git/")
+  chart                      = try(var.volcano_helm_config["chart"], "volcano")
+  version                    = try(var.volcano_helm_config["version"], "1.5")
+  timeout                    = try(var.volcano_helm_config["timeout"], 300)
+  values                     = try(var.volcano_helm_config["values"], null)
```

Comment:
Do we need to specify any default values for deploying this Helm chart?

Reply:
Did not seem so I run it successfully in an EKS cluster from the test folder of this repo had to change the repo to this: "https://volcano-sh.github.io/helm-charts"

And here is my run outcome: 
terraform-aws-eks-data-addons/test - (main) > kubectl get all -n volcano-system
NAME                                      READY   STATUS      RESTARTS   AGE
pod/volcano-admission-84fd7c8df5-msnbj    1/1     Running     0          2m27s
pod/volcano-admission-init-bhpdt          0/1     Completed   0          2m27s
pod/volcano-controllers-6b7f4c89d-k29gw   1/1     Running     0          2m27s
pod/volcano-scheduler-9c4bf7d4d-vpqsm     1/1     Running     0          2m27s

NAME                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/volcano-admission-service   ClusterIP   172.20.54.247   <none>        443/TCP    2m28s
service/volcano-scheduler-service   ClusterIP   172.20.217.35   <none>        8080/TCP   2m28s

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/volcano-admission     1/1     1            1           2m28s
deployment.apps/volcano-controllers   1/1     1            1           2m28s
deployment.apps/volcano-scheduler     1/1     1            1           2m28s

NAME                                            DESIRED   CURRENT   READY   AGE
replicaset.apps/volcano-admission-84fd7c8df5    1         1         1       2m29s
replicaset.apps/volcano-controllers-6b7f4c89d   1         1         1       2m29s
replicaset.apps/volcano-scheduler-9c4bf7d4d     1         1         1       2m29s

NAME                               COMPLETIONS   DURATION   AGE
job.batch/volcano-admission-init   1/1           9s         2m30s

and if you compare with the link everything looks the same Step: 2
Use Helm charts to install Volcano:
https://volcano.sh/en/docs/installation/